pkgs/development/python-modules/coconut/default.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildPythonApplication,
+  fetchPypi,
+  fetchpatch,
+
+  dos2unix,
+  findutils,
+
+  cPyparsing,
+  ipykernel,
+  mypy,
+  pygments,
+  prompt_toolkit,
+  watchdog
+}:
+
+buildPythonApplication rec {
+  pname = "coconut";
+  version = "1.4.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "02rccpjnfxafp7csh49niba239yggp18cgjd2n9agalg71vfhlsh";
+  };
+
+  nativeBuildInputs = [ dos2unix findutils ];
+  propagatedBuildInputs = [ cPyparsing pygments prompt_toolkit ipykernel mypy watchdog ];
+
+  prePatch = "find . -name '*.py' -print0 | xargs -0 dos2unix";
+  patches = [
+    (fetchpatch {
+      name = "fix-setuptools-version-check.patch";
+      url = "https://github.com/LibreCybernetics/coconut/commit/2916a087da1e063cc4438b68d4077347fd1ea4a2.patch";
+      sha256 = "136jbd2rvnifw30y73vv667002nf7sbkm5qyihshj4db7ngysr6q";
+    })
+  ];
```

Comment:
Missing meta section.

Reply:
Ah right, forgot to commit, will amend shortly.